templates/auth.conf.erb

```diff
@@ -84,11 +84,13 @@
 path /puppet/v3/environments
 method find
 allow *
+<% if @puppetversion.to_f < 5.0 -%>
```

Comment:
No, it's gone: https://docs.puppet.com/puppetserver/latest/release_notes.html#deprecation-removed-resourcetypes-api-endpoints-and-resourcetype-cli-face

However, this auth.conf file here is not used for puppetserver anyway now: https://docs.puppet.com/puppetserver/latest/release_notes.html#deprecation-legacy-authconf-settings-are-no-longer-enabled-by-default